app/src/main/java/com/example/aaa/pinkcalculator/Numbers.java

```diff
@@ -1,65 +1,69 @@
 package com.example.aaa.pinkcalculator;
 
 public class Numbers {
+    private static final String NULL = "0";
+    private static final String POINT = ".";
+    private static final String MINUS = "-";
```

Comment:
As you have the same constants in `MainActivity`, you can extract it into another file with constants.